exercise32.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+import random
+
+def random_int_level_1():
+    """Generate a random number for level 1"""
+
+    return random.randrange(0, 10)
+
+def response_to_user_guess_too_high(user_guess):
```

Comment:
So these three functions shouldn't do anything with the numbers, they should just call the `print` function...